src/services/users/reset-password/reset-password.test.js

```diff
@@ -0,0 +1,24 @@
+const chai = require('chai')
+const verifyResetToken = require('./verifyPasswordResetToken')
+const generateAccessToken = require('../generate-access-token')
+
+const testEmail = 'testemail@gmail.com'
+
+        
+const { expect } = chai
+
+describe('verifyPasswordResetToken', () => {
+	it('should throw an error if the token is invalid', () => {
+		const token = 'invalid-token'
+		expect(() => verifyResetToken(token)).to.throw()
+	})
+})
+describe('verifyPasswordResetToken', () => { 
```

Comment:
just one` describe` block hosting the two `it` blocks is fine. In cases there needs to be a different one or inner describe, it shouldn't have the same description as well